app/api/helpers/jwt.py

```diff
@@ -18,6 +18,8 @@ def jwt_authenticate(email, password):
     if user is None:
         return None
     auth_ok = user.facebook_login_hash == password or user.is_correct_password(password)
+    if auth_ok and user.is_blocked:
+        return "spam"
```

Comment:
Revert

Reply:
why?

Reply:
I am checking spam after login is successfull. is this logic wrong too?

Reply:
Yes, the function returns User or None, not String. It breaks the contract of the function and will break the callers elsewhere. You never modify the signature of function to something else so easily. What if some caller calls this function and checks if return is not None, and expects the return type to be User like it previously was? Now, the server will crash because it is not user, it is String!

Reply:
ok 🙄